chapter_convolutional-modern/alexnet_vn.md

```diff
@@ -103,7 +118,12 @@ In the case of an image, the lowest layers might come to detect edges, colors, a
 Indeed, :cite:`Krizhevsky.Sutskever.Hinton.2012` proposed a new variant of a convolutional neural network which achieved excellent performance in the ImageNet challenge.
 -->
 
-*dịch đoạn phía trên*
+Một nhóm nhà nghiên cứu khác, bao gồm Yann LeCun, Geoff Hinton, Yoshua Bengio, Andrew Ng, Shun-ichi Amari, và Juergen Schmidhuber, đã có những kế hoạch khác. 
+Họ tin rằng các đặc trưng tự chúng có thể được học. 
+Hơn nữa, họ cũng tin rằng với một độ phức tạp hợp lý, các đặc trưng có thể được phân thành thứ lớp với nhiều tầng học cùng nhau, mỗi tầng có các tham số huấn luyện có thể học được. 
```

Comment:
```suggestion
Hơn nữa, họ cũng tin rằng để đạt được một độ phức tạp hợp lý, các đặc trưng nên được phân thành thứ lớp với nhiều tầng học cùng nhau, mỗi tầng có các tham số có thể được huấn luyện. 
```

Reply:
ở đây mình không nghĩ là đạt được mà bản thân data đã chứa thông tin với một độ phức tạp nào đó đủ để ta collect, một cách tự nhiên nó nói lên nó là ai mà không cần đến suy diễn của cách ta mô hình

Reply:
Chỗ này thì em ko đồng ý, mình đang hiểu nguyên nhân - kết quả ngược nhau rồi. Em thấy hình như ý của anh thiếu chữ `ought to`.

 Ở đây tiếp sau ý `nên học đặc trưng`. Vậy nên **để học các biểu diễn đủ độ phức tạp**, mình phải xắp sếp các tầng là có phân cấp.

Với một bộ dữ liệu đủ độ phức tạp thì các đặc trưng cũng đâu tự phân cấp với nhau được.

Reply:
Vì các đặc trưng bản thân tự nó có trong dữ liệu, nên mục đích ở đây không phải là tạo ra các đặc trưng phức tạp. Mà dữ liệu đã chứa các đặc trưng phức tạp ẩn, hay tường minh sẵn rồi. Việc cần làm của máy học là huấn luyện để nhớ các đặc trưng đó

Reply:
Cho nên ở câu cuối của đoạn tác giả nói là ở những tầng thấp nhất có thể học được các đặc trưng cơ bản như màu sắc, đường biên. Rõ ràng bất cứ ảnh nào cũng có sẵn cái này mà không phải do ta cố tình tạo ra cho phức tạp. Máy có chức năng học các đặc điểm này.

Reply:
> Vì các đặc trưng bản thân tự nó có trong dữ liệu

Mình hiểu khác nhau chỗ này. Thông tin thì mới tự có trong dữ liệu, còn các đặc trưng hay biểu diễn của dữ liệu ở đây là trích xuất được từ các tầng xếp lên nhau. Cùng tập dữ liệu nhưng chỉ cho học đặc trưng qua 1 tầng ẩn thì ko thể đạt được các biểu diễn đủ độ phức tạp, vậy nên mình mới phải phân lớp đặc trưng thành nhiều tầng.

Reply:
"Để đạt được độ phức tạp" khiến mình từ vị trí người đọc thấy cách làm này rất chủ quan và không tự nhiên.

Reply:
Em thì ko thấy vấn đề gì nhưng ko biết a thấy vầy thì sao?
```suggestion
Hơn nữa, họ cũng tin rằng để có được độ phức tạp hợp lý, các đặc trưng nên được phân thành thứ lớp với nhiều tầng học cùng nhau, mỗi tầng có các tham số có thể được huấn luyện. 
```

Reply:
mình không hiểu tại sao lại có từ "thứ lớp" ở đây nhỉ?

Reply:
Từ chữ hierarchically đó @ngcthuong :)

Reply:
```suggestion
Hơn nữa, họ cũng tin rằng với một độ phức tạp hợp lý, các đặc trưng có thể được phân thành thứ lớp với nhiều tầng học cùng nhau, mỗi tầng có các tham số huấn luyện có thể học được. 
```

Bản thân từ nhiều tầng nó đã thể hiện ý nghĩa của từ đó "hierarchicaly" rồi mà nhỉ?